go-controller/pkg/ovndbmanager/ovndbmanager.go

```diff
@@ -98,8 +105,19 @@ func ensureLocalRaftServerID(db string) {
 	out, stderr, err = appCtl("cluster/status", dbName)
 	if err != nil {
 		klog.Warningf("Unable to get cluster status for: %s, stderr: %v, err: %v", db, stderr, err)
+		if *clusterStatusRetryCnt > maxClusterStatusRetry {
```

Comment:
Also, the ticker executes these functions every 1 minute. Which means that with `maxClusterStatusRetry` set to 10, we wait ~ 10 minutes to exit....feels like this can be done after fewer retries and quicker?

Reply:
I debated if we need to do this quicker. But also wanted to give enough time for other master pods to come up and join/form the cluster. The cluster/status check is done twice and hence for every minute the counter will bump by 2 and we will hit the abort button after 5 minutes. This is all pulling counters and values out of thin-air really.

Reply:
> If it's done for convenience, then maybe just perform this check in one of the functions?

It's not done for convenience only. This check in `ensureLocalRaftServerID` looks for the sid with the same address as the node to match the server id returned by `ovsdb-tool db-sid` command. If there is a sid found with the same address that's different than the one returned by the db-sid command, we boot that sid from the raft cluster.

In case of the `ensureClusterRaftMembership` it is used to remove any unknown members from raft compared to a list given to us by `GetPods()`. 

Both the checks are needed.

Reply:
there could be some optimization there, but I'm OK with how it is now with sync lib. @alexanderConstantinescu if you think it needs more changes let us know. I'll refrain from merging until we resolve this one.

Reply:
I guess I am OK with it too, given the urgency to get this patch in. It's just that I don't understand the point of doing it twice like this, and I just wanted a clarification of that

Reply:
Yeah, the only thing there we can do is parse some data between the two functions by doing a `cluster/status` outside. I don't think it's going to get us that much TBH.

Reply:
@trozet : this patch can be merged. I've discussed this with @abhat and we all agree whatever improvements can be done here, can also wait.